TAMPA-LEARNERS(04-2018).md

```diff
@@ -48,3 +48,4 @@
 - [Corrao, Paul]()
 - [Alvarez, Raimir]()
 - [McFarquhar, Colin]()
+- [Johnson, Barry]()
```

Comment:
Hey, Barry, you need to put a link to your GitHub repository. Thanks!

Reply:
Hey Maria, I added a second commit with a link to my GitHub repo. Thanks for letting me know!